addons/nuxeo-datadog-reporter/src/main/java/org/nuxeo/datadog/reporter/NuxeoDatadogReporter.java

```diff
@@ -1,26 +1,61 @@
-package org.coursera.metrics.datadog;
+/*
+ * (C) Copyright 2020 Nuxeo (http://nuxeo.com/) and others.
```

Comment:
We can't put our copyright or license here. Please just copy https://github.com/coursera/metrics-datadog/blob/master/LICENSE into the header of this file. Then we can add just "Additional contributors: Nuxeo (Benoit Delbosc)" after it